lib/temporal/configuration.rb

```diff
@@ -31,7 +31,12 @@ class Configuration
       # See       # https://docs.temporal.io/blog/activity-timeouts/#schedule-to-start-timeout
       schedule_to_start: nil,
       start_to_close: 30,     # Time spent processing an activity
-      heartbeat: nil          # Max time between heartbeats (off by default)
+      heartbeat: nil,         # Max time between heartbeats (off by default)
+      # If a heartbeat timeout is specified, 80% of that value will be used for throttling. If not specified, this
+      # value will be used.
+      default_heartbeat_throttle_interval: 30,
```

Comment:
are these the same as Java / Go and if not, how did you arrive at these!

Reply:
These came from the Go SDK. I believe they are the same in Java.

Reply:
Yeah, mentioning the provenance in the code would be great.